lib/nolibc/include/fcntl.h

```diff
@@ -162,6 +162,9 @@ struct f_owner_ex {
 
 #define FALLOC_FL_KEEP_SIZE 1
 #define FALLOC_FL_PUNCH_HOLE 2
+#define FALLOC_FL_COLLAPSE_RANGE 8
+#define FALLOC_FL_ZERO_RANGE 0x10
```

Comment:
Adding comment here so we don't forget: This is defined by kernel headers not libc's. So native builds with musl for example would fail at build time since it does not define these.